### PR TITLE
add type definition for parsimmon

### PR DESCRIPTION
**Pull request by mizunashi-mana**
Reference:
https://github.com/jneen/parsimmon#adding-base-parsers

### CODE REVIEW

**Comment by mizunashi-mana** [comment]
Sorry for reviewer.
I made a big mistake, and fixed.

File: parsimmon/parsimmon.d.ts [hunk 1]
```diff
@@ -103,6 +105,14 @@ declare module 'parsimmon' {
 		export function seq<U>(...parsers: Parser<U>[]): Parser<U[]>;
 		export function seq(...parsers: Parser<any>[]): Parser<any[]>;
 
+		export type successFunctionType<U> = (index: number, result: U) => Result<U>;
```
Comment (vvakame) [inline parsimmon/parsimmon.d.ts]: please use CamelCase.

**Comment by mizunashi-mana** [comment]
Oh, sorry.
I fixed to use CamelCase.

**Comment by horiuchi** [comment]
@mizunashi-mana Thanks mate!

**Pull request merged**